tools/distpartitioning/dataset_utils.py

```diff
@@ -3,7 +3,102 @@
 import constants
 import torch
 
-def get_dataset(input_dir, graph_name, rank, num_node_weights):
+def get_ci_nodefeats(rank, feat_name):
+    '''
+    Function to generate dummpy node features for the CI test graph, which
+    is described in constants.py in detail. 
+
+    Parameters:
+    -----------
+    rank : int
+        unique integer assigned to each process in a distributed execution.
+    feat_name : string
+        this string is used as feature name 
+
+    Returns:
+    --------
+    numpy ndarray : 
+        an array of pre-defined dimensions with predetermined data, based on 
+        the rank of the process
+    '''
+    if (feat_name == "nodes-feat-01"):
+        return np.ones((10,20), dtype=np.float)*(rank+1)
```

Comment:
it's better to not use value 1 as node features. if you mess up the node IDs, you cannot tell from the node features. you can use either sequence numbers or random numbers to distinguish nodes and edges.